src/osm_configurator/view/states/positioned_frame.py

```diff
@@ -10,42 +16,92 @@ class PositionedFrame:
     it is shown on a Window.
     """
 
-    def __init__(self, frame, colum, line, state_manager, control):
+    def __init__(self, frame: TopLevelFrame, colum: int, row: int, colum_span: int, row_span: int, sticky: str):
         """
         This method creates a PositionedFrame, which is a Frame and coordinates to its Position.
 
         Args:
             frame (top_level_frame.TopLevelFrame): The frame you want to give a position
-            colum (int): The Colum the Frame shall be placed in
-            line (int): The Line the Frame shall be placed in
-            state_manager (state_manager.StateManager): The StateManager the frame will use to change states if needed
-            control (control_interface.IControl): The control that a frame shall call, if it needs access to the model
+            colum (int): The Colum the Frame shall be placed in, can't be negative
+            row (int): The Line the Frame shall be placed in, can't be negative
+            colum_span (int): The colum span the frame shall have
+            row_span (int): The row span the frame shall have
+            sticky (str): The stick Type of how the frame is placed in a grid
         """
-        pass
 
-    def get_frame(self):
+        # Testing for correct Types, except for the frame, because Python is a little bitch
+        if not isinstance(colum, int):
```

Comment:
you don't have to do this, this is done automatically

Reply:
I keep my checks for correct types